app/(funnel)/target-score/page.tsx

```
@@ -0,0 +1,56 @@
+  const router = useRouter();
+
+  const handleSubmit = async () => {
+    try {
+      setIsLoading(true);
+      const response = await fetch('/api/target-gpa', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          targetGpa: parseFloat(score),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)